### PR TITLE
Rename "Element Types" to "Element Classes" distinguished from "Eleme…

### PR DESCRIPTION
**Pull request by ZiyunShang**
…ntType"

### Purpose

Rename Element Types node to make it more meaningful and distinguished from "ElementType"
Related to issue https://github.com/DynamoDS/DynamoRevit/issues/2587 

### Declarations

Check these if you believe they are true

- [ ] The code base is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] Snapshot of UI changes, if any.

### Reviewers

@ShengxiZhang @wangyangshi 

### FYIs

@Amoursol @QilongTang @mjkkirschner

### CODE REVIEW

**Review by wangyangshi — APPROVED** [review]

File: src/Libraries/RevitNodesUI/RevitDropDown.cs [hunk 1]
```diff
@@ -947,7 +947,7 @@ public class SpacingRuleLayouts : EnumAsInt<SpacingRuleLayout>
         public SpacingRuleLayouts(IEnumerable<PortModel> inPorts, IEnumerable<PortModel> outPorts) : base(inPorts, outPorts) { }
     }
 
-    [NodeName("Element Types")]
+    [NodeName("Element Classes")]
```
Comment (ZiyunShang) [inline src/Libraries/RevitNodesUI/RevitDropDown.cs:950]: This class is subclass of AllChildrenOfType<Element>, and can't modify its output name.
The output name is set in DynamoCore - https://github.com/DynamoDS/Dynamo/blob/master/src/Libraries/CoreNodeModels/Enum.cs#L65 .
So I think I need submit a update here.

Comment (ZiyunShang) [reply src/Libraries/RevitNodesUI/RevitDropDown.cs:950]: ElementClass
Hi @ShengxiZhang , with this PR, the results as the screenshot shown. We can't rename the output name "Type" on DynamoRevit side.

**Comment by mjkkirschner** [comment]
I think it's important that before making this change testing is done on graphs and custom nodes that reference these nodes to make sure they open okay.

**Comment by ZiyunShang** [comment]
@mjkkirschner I just run all DynamoRevit tests and all have passed. I also test the old script with this change, it can run without error or warning.
test

**Comment by chuongmep** [comment]
I think we not necessary change name to class because engineer difficult for read document update .

**Comment by ZiyunShang** [comment]
> I think we not necessary change name to class because engineer difficult for read document update .

But this does mislead some users and confuse them like https://github.com/DynamoDS/DynamoRevit/issues/2587 . So I think this will be helpful for distinguishing Element Class and ElementType.

**Pull request merged**